scripts/exglobal_stage_ic.sh

```diff
@@ -106,7 +106,7 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
 
     # Ocean Perturbation Files
     # Extra zero on MEMDIR ensure we have a number even if the string is empty
-    if (( 0${MEMDIR:3} > 0 )) && [[ "${OCN_ENS_PERTURB_FILES:-false}" == "true" ]]; then
+    if (( 0${MEMDIR:3} > 0 )) && [[ "${STAGE_OCN_PERTURB_FILES:-false}" == "true" ]]; then
```

Comment:
```suggestion
    if (( MEMBER > 0 )) && [[ "${STAGE_OCN_PERTURB_FILES:-false}" == "true" ]]; then
```
Use `MEMBER` and update the comment on the previous line.
`MEMBER` is set in https://github.com/NOAA-EMC/global-workflow/blob/0edbdc197441582f9c402da0f3a7f144b88960c7/ush/forecast_predet.sh#L121